### PR TITLE
Add [junos_lacp_interfaces] module implementation

### PR DESCRIPTION
**Pull request by ganeshrn**
*  Module implementation with merge, delete, replace, override
*  Implement gathering lacp_interfaces facts

### CODE REVIEW

File: module_utils/network/junos/config/lacp_interfaces/lacp_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+created
+"""
+
+from ansible.module_utils.network.common.cfg.base import ConfigBase
+from ansible.module_utils.network.common.utils import to_list, get_xml_conf_arg
+from ansible.module_utils.network.junos.facts.facts import Facts
+from ansible.module_utils.network.junos.junos import locked_config, load_config, commit_configuration, discard_changes, tostring
+from ansible.module_utils.network.junos.utils import utils as junosutils
+
+import q
```
Comment (danielmellado) [inline module_utils/network/junos/config/lacp_interfaces/lacp_interfaces.py:20]: I guess this is not inteded to be imported in the final revision ;)

**Review by danielmellado — CHANGES_REQUESTED** [review]
Just some minor leftover there

File: module_utils/network/junos/config/lacp_interfaces/lacp_interfaces.py (hunk 1 above)
Comment (danielmellado) [reply module_utils/network/junos/config/lacp_interfaces/lacp_interfaces.py:20]: Besides that, we should deff agree and merge on tox.ini flake8 validations for this repo. IMHO we should just match ansible/ansible for now

**Comment by ganeshrn** [comment]
Moved to https://github.com/ansible/ansible/pull/59708

**Pull request closed without merging**